dbt-redshift/src/dbt/adapters/redshift/connections.py

```diff
@@ -134,6 +139,7 @@ class RedshiftCredentials(Credentials):
     role: Optional[str] = None
     sslmode: UserSSLMode = field(default_factory=UserSSLMode.default)
     retries: int = 1
+    retry_all: bool = False
```

Comment:
I'm wondering if retry_all is obscuring what exactly. Should we say `retry_all_errors` so uses know we aren't retrying failures?

Reply:
yea I went with `retry_all` as that's what we do for spark and snowflake, I think consistency is worth it here

Reply:
Fair I'll buy consistency